sdk/src/wallet/account/operations/retry.rs

```diff
@@ -119,8 +119,9 @@ where
                 // After we checked all our reattached blocks, check if the transaction got reattached in another block
                 // and confirmed
                 if conflicting {
-                    let included_block = self.client().get_included_block(transaction_id).await?;
-                    return Ok(included_block.id());
+                    if let Ok(included_block) = self.client().get_included_block(transaction_id).await {
```

Comment:
Is there a specific error that should end up in `Error::TangleInclusion`? Aren't we ignoring a lot of potentially unrelated error that maybe should be propagated?

Reply:
Don't think there can be a lot of potentially unrelated errors, but changed it